tests/sqllogictests/suites/query/window_function/named_window_basic.test

```diff
@@ -1,278 +1,293 @@
+CREATE TABLE t1(a int)
+
+statement ok
+INSERT INTO t1 VALUES (1),(1),(1),(3),(3),(5),(5)
+
+# named window
+query TIIR
+SELECT depname, empno, salary, sum(salary) OVER w FROM empsalary WINDOW w AS (PARTITION BY depname ORDER BY empno) ORDER BY depname, empno
+----
+develop 7 4200 4200
+develop 8 6000 10200
+develop 9 4500 14700
+develop 10 5200 19900
+develop 11 5200 25100
+personnel 2 3900 3900
+personnel 5 3500 7400
+sales 1 5000 5000
+sales 3 4800 9800
+sales 4 4800 14600
+
+# sum
+query R
+SELECT sum(salary) OVER w ss FROM empsalary WINDOW w AS (PARTITION BY depname ORDER BY salary) ORDER BY depname, ss
+----
+4200
+8700
+19100
+19100
+25100
+3500
+7400
+9600
+9600
+14600
+
+# row_number
+query I
+SELECT row_number() OVER w1 rn1, dense_rank() OVER w2 rn2 FROM empsalary WINDOW w1 AS (PARTITION BY depname ORDER BY salary), w2 AS (PARTITION BY depname ORDER BY salary DESC) ORDER BY depname, rn1
+----
+1 4
+2 3
+3 2
+4 2
+5 1
+1 2
+2 1
+1 2
+2 2
+3 1
+
+# row_number with inherited window
+query I
+SELECT row_number() OVER w1 rn1, dense_rank() OVER w2 rn2 FROM empsalary WINDOW w1 AS (PARTITION BY depname), w2 AS (w1 ORDER BY salary DESC) ORDER BY depname, rn1
+----
+1 2
+2 4
+3 3
+4 1
+5 2
+1 2
+2 1
+1 1
+2 2
+3 2
+
+# sum with inherited window
+query IIII
+select a, sum(a) over w, sum(a) over w1, sum(a) over w2 from t1 window w as (partition by a), w2 as (w1 rows current row), w1 as (w order by a) order by a
+----
+1 3 3 1
+1 3 3 1
+1 3 3 1
+3 6 6 3
+3 6 6 3
+5 10 10 5
+5 10 10 5
+
+# rank_dense
+query TII
+SELECT depname, salary, dense_rank() OVER w FROM empsalary WINDOW w AS (PARTITION BY depname ORDER BY salary) order by depname, salary
+----
+develop 4200 1
+develop 4500 2
+develop 5200 3
+develop 5200 3
+develop 6000 4
+personnel 3500 1
+personnel 3900 2
+sales 4800 1
+sales 4800 1
+sales 5000 2
+
+# rank
+query TII
+SELECT depname, salary, rank() OVER w FROM empsalary WINDOW w AS (PARTITION BY depname ORDER BY salary) order by depname, salary
+----
+develop 4200 1
+develop 4500 2
+develop 5200 3
+develop 5200 3
+develop 6000 5
+personnel 3500 1
+personnel 3900 2
+sales 4800 1
+sales 4800 1
+sales 5000 3
+
+
+# row_number
+query I
+SELECT row_number() OVER w rn FROM empsalary WINDOW w AS (PARTITION BY depname ORDER BY salary) ORDER BY depname, rn
+----
+1
+2
+3
+4
+5
+1
+2
+1
+2
+3
+
+# rank_dense
+query TII
+SELECT depname, salary, dense_rank() OVER w FROM empsalary WINDOW w AS (PARTITION BY depname ORDER BY salary) order by depname, salary
+----
+develop 4200 1
+develop 4500 2
+develop	5200 3
+develop	5200 3
+develop 6000 4
+personnel 3500 1
+personnel 3900 2
+sales 4800 1
+sales 4800 1
+sales 5000 2
+
+# rank
+query TII
+SELECT depname, salary, rank() OVER w FROM empsalary WINDOW w AS (PARTITION BY depname ORDER BY salary) order by depname, salary
+----
+develop 4200 1
+develop 4500 2
+develop 5200 3
+develop 5200 3
+develop 6000 5
+personnel 3500 1
+personnel 3900 2
+sales 4800 1
+sales 4800 1
+sales 5000 3
+
+# percent_rank
```

Comment:
only add this test.
no idea why lead to so many diff.

Reply:
![image](https://user-images.githubusercontent.com/5782159/232397549-323cbdb9-7735-4494-a098-7c417c1e79cc.png)

Reply:
Maybe the file encoding format is changed?